### PR TITLE
Ghosts look like ghosts again 

### PR DESCRIPTION
**Pull request by Robustin**
This is not a meme

I repeat

This is not a meme

Ghosts should be spooky, not look like they're off to a rave.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
It's a pref toggle. Turf it off if you dislike it.

**Comment by GunHog** [comment]
Like Kor said, no one is forcing this on you. Turn it off, and move on.

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -205,9 +205,14 @@ var/list/image/ghost_images_simple = list() //this is a list of all ghost images
...
 	if(b_val > 255)
 		b_val = 255
-
+	
 	return num2hex(r_val, 2) + num2hex(g_val, 2) + num2hex(b_val, 2)
 
+/mob/proc/whiten_color() 
+	var r_val = 246
+	var g_val = 246
+	var b_val = 246
```
Comment (phil235) [inline code/modules/mob/dead/observer/observer.dm]: This doesn't use the correct formatting (should var/r_val ).
The proc does literally nothing.

Comment (GunHog) [inline code/modules/mob/dead/observer/observer.dm]: You mean that he did not bother to test it? All the more reason to shut this terrible PR down!

**Comment by ChangelingRain** [comment]
Turn off the pref, that's the solution here
The pref doesn't work? Fix it!

**Comment by Robustin** [comment]
There's no option to have hair style but no color.

My immersions are literally being eviscerated.

File: code/modules/mob/dead/observer/observer.dm (hunk 1 above)
Comment (duncathan) [inline code/modules/mob/dead/observer/observer.dm]: Actually uh this formatting works...

Comment (phil235) [inline code/modules/mob/dead/observer/observer.dm]: The formatting works but it's not the standard we're using for tg.

@GunHog You don't need to test anything to see that this code does nothing, it's really obvious.

**Pull request closed without merging**